viewer/viewer-awt/src/main/java/org/icepdf/ri/common/utility/annotation/properties/BorderPanel.java

```diff
@@ -94,9 +94,9 @@ public void setAnnotationComponent(AnnotationComponent newAnnotation) {
 
         // apply annotation values.
         if (annotation.getLineThickness() == 0) {
-            applySelectedValue(linkTypeBox, Annotation.INVISIBLE_RECTANGLE);
+            applySelectedValue(linkTypeBox, false);

```

Comment:
Any chance you can also rename 'linkTypeBox' to 'borderTypeBox'  or something similar while your in this class.

Reply:
I changed it to borderVisibleBox, if that's okay for you

Reply:
Awesome and thanks.